### PR TITLE
Export endpoint descriptions from READMEs

### PR DESCRIPTION
**Pull request by austinborn**
## Description
Add description as exported attribute for endpoints with README details.

## Changes
- Add `export const description = '...'` to endpoint files
- Remove descriptions from `schema/env.json` in cases where the top-level description was taken from specific endpoint descriptions (since there wasn't an alternative at the time)

## Steps to Test
Generic tests (`yarn test:unit`, `yarn test:integration`)

## Quality Assurance

- [x] If a new adapter was made, or an existing one was modified so that its environment variables have changed, update the relevant `<ADAPTER_PACKAGE>/schemas/env.json` and `<ADAPTER_PACKAGE>/README.md`
- [x] If a new adapter was made, or an existing one was modified so that its environment variables have changed, update the relevant `infra-k8s` configuration file.
- [x] The branch naming follows git flow (`feature/x`, `chore/x`, `release/x`, `hotfix/x`, `fix/x`) or is created from Clubhouse/Shortcut
- [x] This is related to a maximum of one Clubhouse/Shortcut story or GitHub issue
- [x] Types are safe (avoid TypeScript/TSLint features like any and disable, instead use more specific types)

### CODE REVIEW

File: packages/sources/1forge/src/endpoint/quotes.ts [hunk 1]
```diff
@@ -11,6 +11,9 @@ import { NAME as AdapterName } from '../config'
 export const supportedEndpoints = ['quotes', 'forex', 'price']
 export const batchablePropertyPath = [{ name: 'base' }, { name: 'quote' }]
 
+export const description =
+  'Returns a batched price comparison from a list currencies to a list of other currencies.\n\n[`/quotes`](https://1forge.com/api#quotes) - Convert from one currency to another.\n\n**NOTE: the `price` endpoint is temporarily still supported, however, is being deprecated. Please use the `quotes` endpoint instead.**'
+
```
Comment (alejoberardino) [inline packages/sources/1forge/src/endpoint/quotes.ts:16]: I'd recommend using backticks here and other descriptions with newlines, it'd make it cleaner imo

Comment (austinborn) [reply packages/sources/1forge/src/endpoint/quotes.ts:16]: How would you format this one for example? Here's my attempt at it:
```suggestion
export const description =
`Returns a batched price comparison from a list currencies to a list of other currencies.

[\`/quotes\`](https://1forge.com/api#quotes) - Convert from one currency to another.
**NOTE: the \`price\` endpoint is temporarily still supported, however, is being deprecated. Please use the \`quotes\` endpoint instead.**`
```

Comment (alejoberardino) [reply packages/sources/1forge/src/endpoint/quotes.ts:16]: Yeah, that's great imo 🙂

Comment (austinborn) [reply packages/sources/1forge/src/endpoint/quotes.ts]: Ok, all descriptions that used newline characters have been updated with this pattern

**Review by alejoberardino — APPROVED** [review]

**Pull request merged**